### PR TITLE
Some minor fixes for warnings while running the tests

### PR DESCRIPTION
**Pull request by duglin**
Just some minor things I noticed while running the tests today - nothing big.

Signed-off-by: Doug Davis <dug@us.ibm.com>

### CODE REVIEW

File: cmd/kubeadm/app/phases/certs/certs.go
Comment (duglin) [inline cmd/kubeadm/app/phases/certs/certs.go]: this one complained about having `%s` but not using Infof

File: pkg/util/ipset/ipset.go
Comment (duglin) [inline pkg/util/ipset/ipset.go]: too few %s's

File: pkg/volume/azure_dd/azure_dd.go
Comment (duglin) [inline pkg/volume/azure_dd/azure_dd.go]: wrong datatype on the %'s - they're numeric not string

**Comment by duglin** [comment]
easy review
ping @detiber @islinwb @dims

File: pkg/volume/azure_dd/azure_dd.go
Comment (linyouchong) [reply pkg/volume/azure_dd/azure_dd.go]: We should keep the first `%s`, *size.Name is string.

**Review by neolit123 — COMMENTED** [review]
/kind cleanup

File: pkg/util/ipset/ipset.go
Comment (islinwb) [reply pkg/util/ipset/ipset.go]: https://github.com/kubernetes/kubernetes/pull/68263 fixes this but it's ok.

File: pkg/volume/azure_dd/azure_dd.go
Comment (duglin) [reply pkg/volume/azure_dd/azure_dd.go]: `%v` works just fine for strings, but I'll switch it back to `%s`.
I don't think it was a linter error, is linter run as part of "make test" ?
The console showed this:
```
# k8s.io/kubernetes/cmd/kubeadm/app/phases/certs
cmd/kubeadm/app/phases/certs/certs.go:113: Verbose.Infoln call has possible formatting directive %s
```

**Comment by duglin** [comment]
addressed comment

File: pkg/volume/azure_dd/azure_dd.go
Comment (neolit123) [reply pkg/volume/azure_dd/azure_dd.go]: > is linter run as part of "make test"

i don't think so.

**Comment by neolit123** [comment]
thanks
/lgtm

**Comment by duglin** [comment]
ping @neolit123

**Comment by neolit123** [comment]
@duglin thanks for the ping.
one of the milestone maintainers should add a milestone if this is to be merged for 1.12.
otherwise it has to wait after 1.12 - this is more likely given it requires approval by multiple parties.

/assign @fabriziopandini

**Comment by duglin** [comment]
sounds fair, definitely not critical for 1.12

**Comment by neolit123** [comment]
/assign @fabriziopandini @timothysc 
PR is LGTM, needs approve.

**Review by timothysc — APPROVED** [review]
/approve

**Comment by duglin** [comment]
any chance of getting this one in?

**Review by andyzhangx — APPROVED** [review]
/lgtm
/approve

**Review by timothysc — APPROVED** [review]
/lgtm 
/approve

**Comment by duglin** [comment]
@brendanburns any chance you can review this? This minor PR has been there for a while now and I'd like to get it into 1.13

**Comment by duglin** [comment]
all tests are passing - who can push this over the finish line?

**Pull request merged**
